lib/aur-search

```diff
@@ -121,13 +109,43 @@ case $query_type in
     search) format=${format-short} ;;
 esac
 
-# set filters
+# set parsers
```

Comment:
unrelated change, see also https://en.wikipedia.org/wiki/Filter_(software)

Reply:
if the functions are called `parse` and they are doing parsing, I think the comment should also say that